src/app/(app)/changepassword.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { useState }  from 'react'; 
+import { View , TouchableOpacity , Text , Image , StyleSheet , TextInput } from 'react-native'
+import Icon from 'react-native-vector-icons/FontAwesome'
+
+
+const changepassword = () => {
+  const [oldpassword , Setpassword] = useState('');
+  const [newpassword , Setnewpassword] = useState('');
+  const [confirmpassword , Setconfirmpassword] = useState('');
+  const [SecureTextEntry,SetSecureTextEntry] = useState(true);
+  
+  const toggleSecureTextEntry = () => {
+    SetSecureTextEntry(!SecureTextEntry);
+  };
+
+  return (
+
+    <View style={styles.container}>
+      <View style={styles.header}>
+        <Text style={styles.title}>Change Password</Text>
+      </View>
+    
+    <View style={styles.form}>
+        <View style={styles.inputContainer}>
+          <Icon name="lock"
+          size={20}
+          color="#000" />
+          
+          <TextInput
+            value={oldpassword}
+            placeholder='Old Password'
+            onChangeText={Setpassword}
+            secureTextEntry={SecureTextEntry}
+            style={styles.input}
+          />
```

Comment:
use components already defined in the project for consistency across screens. See components directory and follow how buttons and TextInput has been used in Login / signup screens